getstats-mangle.js

```diff
@@ -252,20 +252,20 @@ module.exports = function(stats) {
           standardReport[newId].sliCount = report.sliCount; // undefined yet
         }
         if (report.remoteSource) {
-          standardReport[newId].type = 'inboundrtp';
+          standardReport[newId].type = 'inbound-rtp';
           standardReport[newId].packetsReceived = report.packetsReceived;
           standardReport[newId].bytesReceived = report.bytesReceived;
           standardReport[newId].packetsLost = report.packetsLost;
         } else {
-          standardReport[newId].type = 'outboundrtp';
+          standardReport[newId].type = 'outbound-rtp';
```

Comment:
I would add some constants for these strings that are repeated in miultiple places

Reply:
naming a variable outboundrtp with a string outbound-rtp is less readable.